src/screens/welcome/WelcomeScreen.jsx

```diff
@@ -8,6 +8,7 @@ import { makeHandlers } from '/libs/functions/makeHandlers'
 import { getColors } from '/ui/colors'
 import { useDimensions } from '/libs/dimensions'
 import { LoginScreen } from '/screens/login/LoginScreen'
+import { useInstallReferrer } from './install-referrer/useInstallReferrer'
```

Comment:
alias?

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/compare/e296534f408c5ca5dce5b8af44df9d6144165350..1b2e03a8f035457424a35b1ed39ad6fbb317bb73